src/utils/validateInputWithRegex.ts

```diff
@@ -0,0 +1,26 @@
+import { EnvVersionError } from "../services/envVersion";
+
+export const validate = (id: string, idRegex: string): boolean => {
+  return id.match(idRegex) !== null;
+};
+
+export const validateAndCleanRegexInput = (
+  inputId: string,
+  idRegex: string | undefined
+): string => {
+  if (!idRegex)
+    throw new EnvVersionError(
+      "Encountered an issue obtaining environment information. We've noted this down and are looking into it!"
+    );
+  const isValid = validate(inputId, idRegex);
+  if (!isValid)
+    throw new Error(
+      "Please check that the identification is in the correct format"
```

Comment:
So here we use the long form instead of id ah haha

Reply:
eh no lets do ID😅

Reply:
haha okk xD